buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -165,7 +165,7 @@
       "alwaysRun": false,
       "displayName": "packages -> dotnet.myget.org",
       "timeoutInMinutes": 0,
-      "condition": "and(succeeded(), eq(variables['PublishToMyGet'], 'True'), eq(variables['PB_ConfigurationGroup'], 'Release'))",
+      "condition": "and(succeeded(), eq(variables['PB_PublishType'], 'myget'), eq(variables['PB_ConfigurationGroup'], 'Release'))",
```

Comment:
Condition should be `and(succeeded(), contains(variables['PB_PublishType'], 'myget'), eq(variables['PB_ConfigurationGroup'], 'Release'))`

Similar for the others.